### ISSUE TITLE
Unable to load on NeoForge 1.20.4

### ISSUE THREAD

**Issue by Erdragh** [kosma/sqlite-jdbc#3]
When trying to start a NeoForge 1.20.4 server with this "Mod" installed, the following error gets logged:
```
[10:53:54.054] [main/ERROR] [loading.LanguageLoadingProvider/LOADING]: Missing language javafml version [25,) wanted by sqlite-jdbc-3.41.2.1+20230506-all.jar, found 2.0
```
Which results in the jar not getting loaded and the classes not being available for connecting with a database.

**Comment by kosma** [comment]
NeoForge is being drunk. For a quick fix, open the mod JAR and edit `*.mods.toml`, and change `loaderVersion` to `[1,)`.

**Comment by Erdragh** [comment]
I've actually already tried this, but that just results in the following: https://gist.github.com/Erdragh/6af95baedf7134d8d9e161a694598537

**Comment by kosma** [comment]
Seems like NeoForge decided to diverge from the Forge way of loading mods, I wouldn't be surprised if it was purely for creating a bigger mess. I'll look into this but I can't promise any kind of timeline :|

**Comment by Erdragh** [comment]
I wouldn't say there was any malicious intent. Also, in versions >= 1.20.5 NeoForge uses `neoforge.mods.toml` instead of just `mods.toml`. Probably to resolve these exact kinds of issues. If there's no easy way to fix this for 1.20.4, I'll just not release a 1.20.4 version of my mod for Neo. Thanks for your quick reply though!

**Comment by kosma** [comment]
I don't think it was malicious intent - I'm simply not a fan of overzealous validation of mod jars - it causes trouble for people trying to do advanced things (or cursed things, as shown here).

Also - I assume that to have the mod loadable by NeoForge 1.20.5+, I have to include `neoforge.mods.toml` as well? The NeoForge documentation doesn't say anything about `neoforge.mods.toml`...

**Comment by kosma** [comment]
Hmm. Another option would be using the `lowcodefml` loader, but then I think their validator will balk :|

**Comment by Erdragh** [comment]
> Also - I assume that to have the mod loadable by NeoForge 1.20.5+, I have to include `neoforge.mods.toml` as well? The NeoForge documentation doesn't say anything about `neoforge.mods.toml`...

The documentation also doesn't have anything on NeoGradle yet. The development is very rapid, which is both a bad and a good thing imo, but that's beside the point.
You can reference [the MDK](https://github.com/neoforged/MDK/blob/main/src/main/resources/META-INF/neoforge.mods.toml) for the neoforge.mods.toml changes
